client/js/screens/LogIn/LogInContainer.js

```diff
@@ -1,9 +1,30 @@
 import React, { Component } from "react";
 import Login from "./Login";
+import gql from "graphql-tag";
+import { Text, View } from "react-native";
+import { Mutation } from "react-apollo";
+
+const LoginMutation = gql`
+  mutation AuthenticateUser($email: String!, $password: String!) {
+    authenticateUser(email: $email, password: $password) {
+      token
+      id
+    }
+  }
+`;
 
 class LoginContainer extends Component {
   render() {
-    return <Login />;
+    return (
+      <Mutation mutation={LoginMutation} >
+        {(authenticateUser, { data }) => (
+          <View>
+            <Login login={authenticateUser} />
+            {console.log("token coming here", data)}
+          </View>
+        )}
+      </Mutation>
+    );
```

Comment:
you should not have images in the screen dir. move them to assets dir if you need them.